public/javascripts/webrtc.js

```diff
@@ -370,24 +370,15 @@ const chatform = document.querySelector('#data');
       console.log('Message: ', message);
       const log = document.querySelector('#chat-log');
       const sender = document.createElement('li');
-      sender.innerText = $peers.names[from];
-      sender.className = 'username-class';
+      sender.innerText = $peers.names[from] + ':';
+      sender.className = 'usernameclass';
       log.appendChild(sender);
       const li = document.createElement('li');
       li.innerText = message;
+      li.className = 'undefined';
```

Comment:
I think using other words instead of 'undefined' for the specific purpose would be better.

Reply:
I think undefined is a fail-safe. So, when you can't properly pass the class name it will result in being labeled undefined. But, yes the name undefined isn't the greatest class name in most cases.